### ISSUE TITLE
User suggestion: include details for reporting gas leaks

### ISSUE THREAD

**Issue by MyfanwyNixon** [mysociety/fixmystreet#501]
Until we report to gas companies as per #16, this user suggests that we should add a note to say that gas leaks should be reported urgently to the National Grid.

We could do this quite easily on http://www.fixmystreet.com/faq, under "Councils often have direct hotlines for these sorts of issues."

Suggested wording: 
"Suspected gas leaks should be reported to the National Grid on 0800 111 999. Calls are free".

**Comment by Gemmamysoc** [comment]
User feedbak: 

"I’d like to help improve FixMyStreet, in particular reporting (non-life-threatening) problems with pylons.

I tried to use FixMyStreet to report a loudly buzzing pylon this morning, and I got the email below back from the local council, rightly saying it’s not theirs but National Grid's responsibility.

Could FixMyStreet tell the user, e.g. in the FAQs, that problems with pylons should be reported to National Grid or their local distribution network operator?"
